x-pack/plugins/session_view/public/components/Application/index.tsx

```diff
@@ -24,7 +24,7 @@ const Links = () => {
       css={css`
         padding: 20px;
         display: flex;
-      `}
+`}
```

Comment:
this indentation doesn't seem right

Reply:
hmm weird that it is what `scripts/stylelint.js` is trying to do, but i will push a fix